socialhome/streams/views.py

```diff
@@ -10,12 +12,67 @@ class BaseStreamView(ListView):
     ordering = "-created"
     paginate_by = 30
     stream_name = ""
+    vue = False
+    queryset = []
```

Comment:
Why has this been added? It's a super bad idea to initialize lists as class attributes. Also, `queryset` is not going to contain a list, but a `QuerySet`.

But I don't see why it was necessary to add this?